### ISSUE TITLE
GraphQL Error when running save-version

### ISSUE THREAD

**Issue by anmiles** [dixahq/kb-typescript-starter#21]
1. I clone `dixahq/kb-typescript-starter`
2. I create `.env` file
3. I change settings: specify correct `accountID` and specify `themeId` (from `npx elevio-kb kb-settings`)
4. I run `npm install`
5. I run `npm run save`. It fails with:

```
Building assets
Building finished, starting copying assets
Copying assets finished
Starting bundling
Finished bundling
GraphQL Error (Code: 500): {"response":{"errors":{"message":"Server Error"},"status":500,"headers":{}},"request":{"query":"query KbThemeBundleUploadUrl($themeId: ID!) {\n  kbThemeBundleUploadUrl(id: $themeId) {\n    bundleHash\n    presignedUrl\n  }\n}","variables":{"themeId":"2"}}}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! elevio-sample@0.2.8 save: `elevio-kb save-version`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the elevio-sample@0.2.8 save script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by noccer** [comment]
Hi @anmiles 

Thanks for reaching out to us and sorry to hear it's not going smoothly for you. We'll try our best to get you up and running asap.

I have taken a look but have not been able to reproduce the issue with the exact steps you provided.

Did you set a your token value in the .env file? The token can be obtained from https://app.elev.io/token.

**Comment by anmiles** [comment]
Here are exact steps that I taken right now and reproduced an error:
- `> git clone git@github.com:dixahq/kb-typescript-starter.git`
- `> cd kb-typescript-starter/`
- `> touch .env`
- insert permanent token from https://app.elev.io/token into .env so its contents look like `TOKEN=[a lot of symbols]`
- add `"themeId": "2"` to settings.json
- change `accountId` in settings.json
- > npm install
- > npm run save
```
Building assets
Building finished, starting copying assets
Copying assets finished
Starting bundling
Finished bundling
GraphQL Error (Code: 500): {"response":{"errors":{"message":"Server Error"},"status":500,"headers":{}},"request":{"query":"query KbThemeBundleUploadUrl($themeId: ID!) {\n  kbThemeBundleUploadUrl(id: $themeId) {\n    bundleHash\n    presignedUrl\n  }\n}","variables":{"themeId":"2"}}}
```

**Comment by anmiles** [comment]
I can confirm that this started reproduce today (probably after recent package release).
2 days ago everything worked perfectly

**Comment by noccer** [comment]
Hey @anmiles 

We've taken a look at this, it looks like the themeId `2` is incorrect. We were wondering: where did you get themeId `2` from?

Can you please confirm if you run `npx elevio-kb kb-settings`, do you get a different themeId number? From a recent internal query I ran, it seems that your themeId is different (I won't reveal it here due to the public nature of this thread).

For context about the 500 error: In the last few days our backend server changed restrictions relating to editing/publishing permissions on themeId `2`, so that is why you are suddenly seeing things not work all of a sudden.

Try update your themeId to the value returned from`npx elevio-kb kb-settings` (assuming it's not `2`)  and let us know if you're still having issues.

Thanks,
Niall

**Comment by anmiles** [comment]
Hi @noccer 
Here is output of `npx elevio-kb kb-settings` on the project with my accountId `621f880b84c76`:
```
{
  kbSettings: {
    canonicalUrl: 'https://wildapricot-test.elevio.help',
    enabled: true,
    id: '1478',
    liveVersion: '[i removed it, not sure if it can be exposed public]',
    siteUrl: 'https://wac-prelive.wildapricot.com',
    themeId: '2'
  }
}
```

**Comment by noccer** [comment]
Hi @anmiles thanks for your patience on this, it was a long weekend for us here in Melbourne.

`themeId: '2'` is the issue here. Theme Id's up to `100` are Elevio reserved and should not be used with custom KB's. As mentioned, our backend server changed restrictions on KB's with themeId 100 and lower. I'm not sure why your KB is on that themeId.

To remedy this we can create your own theme following the instructions here.
https://kb-kit.elevio.help/en/articles/11-deployment#register

tl;dr; try `npx elevio-kb register-theme` and once you run that you should get a new themeId all to yourself to use. Pop this into your settings.json and you should hopefully be able to move forward with your development.

**Comment by anmiles** [comment]
Hi @noccer 
Worked like a charm, thank you!

**Comment by noccer** [comment]
Great to hear. Thanks for sticking with us and good luck building your KB 🚀
